[Develop] - Update documentation on how to upgrade PostgreSQL (#2611)
epiphany-platform/epiphany: docs/home/howto/UPGRADE.md

    epicli prepare --os OS
    ```

>    Where OS should be `centos-7`, `redhat-7`, `ubuntu-18.04`. . This will create a directory called `prepare_scripts` with the needed files inside.

Reviewer: ```suggestion
   Where OS should be `centos-7`, `redhat-7`, `ubuntu-18.04`. This will create a directory called `prepare_scripts` with the needed files inside.
```

- PgPool
- repmgr

### Prerequisites

> The prerequisites below are checked by the preflight script before upgrading PostgreSQL. Never the less its good to

Reviewer: ```suggestion
The prerequisites below are checked by the preflight script before upgrading PostgreSQL. Never the less it's good to
```

check these manually before doing any upgrade:

- Diskspace: When Epiphany upgrades PostgreSQL 10 to 13 it will make a copy of the data directory on each node

>   to ensure easy recovery in the case of a failed data migration. It is up the the user to make sure there is

Reviewer: ```suggestion
  to ensure easy recovery in the case of a failed data migration. It is up to the user to make sure there is
```

>   ennough space available. The used rule is:

Reviewer: ```suggestion
  enough space available. The used rule is:
```

  
  total storage used on the data volume + total size of the data directory < 95% of total size of the data volume

  We use 95% of used storage after data directory copy as some space is needed during the upgrade.

- Cluster health: Before starting the upgrade the state of the PostgreSQL cluster needs to be healthy. This means
  that executing:

  ```shell
  repmgr cluster show
  ```
  

>   Should not fail should not fail and return 0 as exitcode.

Reviewer: ```suggestion
  Should not fail and return 0 as exit code.
```
